pkg/pool-weighted/contracts/lbp/LBPool.sol

```diff
@@ -27,6 +28,12 @@ import { WeightedPool } from "../WeightedPool.sol";
  */
 contract LBPool is WeightedPool, Ownable2Step, BaseHooks {
     using SafeCast for *;
+    address public bootstrapToken;
```

Comment:
We typically make these private and add getters. Apparently there is no way to change these, so they could all be immutable. Instead of individual getters, you could define and implement ImmutableData and DynamicData interfaces (like Weighted and Stable pools do)...

And here we have a little problem, since the WeightedPoolImmutable data is wrong - and it's not virtual. So I think we need to make them virtual and revert (I guess DynamicData is still fine).